facilitator/src/bin/facilitator.rs

```diff
@@ -78,6 +78,23 @@ trait AppArgumentAdder {
     fn add_packet_decryption_key_argument(self: Self) -> Self;
 }
 
+const SHARED_HELP: &str =
+    "Service accounts: Any flag ending in -gcp-sa-email specifies a GCP service account \
+(identified by an email address). Requests to Google Storage (gs://) will be \
+authenticated as this account, by means of impersonation using GKE's Workload \
+Identity feature: https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity.\
```

Comment:
This isn't quite accurate: when using `gs://` buckets, we use GKE Workload Identity no matter what, because that's how we map our Kubernetes service accounts to the per-data share processor GCP SA. The `*-gcp-sa-email` argument specifies a _further_ GCP SA account that we will impersonate once we have obtained the default account's token. Concretely, we use this so that our per-data share processor GCP SA can impersonate the singleton GCP SA that has been blessed by MITRE to write to the sum part buckets.